pages/components/rating/rating.css

```diff
@@ -0,0 +1,48 @@
+
+.rating-star-icon{
+    color: rgba(0, 0, 0, 0.26);
+}
+
+.rating-star-input:enabled .rating-star-icon{
+    cursor: pointer;
+}
+
+.rating-star-input:enabled + .rating-star-icon:hover{
+    color: #ffd675;
+}
+
```

Comment:
I think you should use color with either color variable name or hex color code to maintain consistency